Helper/ConsoleReporter.php

```diff
@@ -57,7 +57,8 @@ public function onAfterRun(CheckInterface $check, ResultInterface $result, $chec
 
         $this->output->write(sprintf(' %s', $check->getLabel()));
 
-        if ($message = $result->getMessage()) {
+        $message = $result->getMessage();
+        if (null !== $message) {
```

Comment:
in theory null should never be returned
https://github.com/zendframework/zend-diagnostics/blob/master/src/Result/ResultInterface.php#L15

@weierophinney: but I suspect that this is just an incorrect docblock?

Reply:
Hi, but even if I modify the docblock in AbstractResult and init the message to empty string by default, then this PR is still needed (empty string evaluates to boolean false).
I can imagine that zend-diagnostics will take forever to merge, but this fix is a quick one.

Reply:
> Would you be willing to create a PR against zend-diagnostics that would initialize the message to an empty string, please, mentioning this issue?

Sure: https://github.com/zendframework/zend-diagnostics/pull/94

Reply:
I guess with your PR to zend diagnostic this change would no longer be necessary?

Reply:
Yes, if it’s going to be accepted. Strictly speaking, the proposed PR in zend is a BC-breaking change and with a very slight chance may break someone else code, so I won’t be surprised if it will be declined in such form.

Any way, thank you for review. I will close this PR and open another one to update zend library as soon as it’s merged and tagged.

Reply:
I'm reviewing it now. :smile: 

In terms of the potential BC breakage:
- It's documented as returning a string.
- None of the shipped result implementations vary from what `AbstractResult` does.
- All reporters assume that method returns a string.

As such, I'm treating it as a bugfix.

Reply:
Just released the patch as version zend-diagnostics 1.3.1.

Reply:
Thank you for review. I will check the rest of the issues if I have time. Actually just intended to use your bundle cause it’s cool but found a minute to contribute. Good luck :)